app-utils/expect/autobuild/patches/0001-fix-build-failure-caused-by-the-pre-C99-syntax-no-lo.patch

```diff
@@ -0,0 +1,292 @@
+From 0a606cbf3bee2e394d95c38f15a92b310469c148 Mon Sep 17 00:00:00 2001
+From: Ilikara <3435193369@qq.com>
+Date: Tue, 6 May 2025 22:14:20 +0800
+Subject: [PATCH] fix build failure caused by the pre-C99 syntax no longer
```

Comment:
Title is for title only - the rest of the details should go into the main commit message.